Makefile

```diff
@@ -15,21 +15,21 @@ check-style: | pre-run node_modules
 
 	npm run check
 
-clean:
+clean: pre-run
 	@echo Cleaning app
 
-	rm -rf node_modules
+	rm -rf node_modules 
 
 pre-run:
 	@echo Make sure no previous build are in the folder
 
-	@rm -rf build/*
+	@rm -rf build/* actions client constants reducers selectors store utils types mattermost.client4* index.* mattermost.websocket_client*
```

Comment:
If we add a new directory, will it be obvious that this needs to updated?

Reply:
This list of folders used to be part of the `make clean`, so I don't think it's a big deal to have to add more folders here